CONTRIBUTING.md

```diff
@@ -87,9 +87,9 @@ Does the same thing as the `test` script, except:
 - Promotes a remote package.
 - Should be run after QA validations.
 
-### `promote:npm:beta` and `promote:npm:latest` scripts
+### `promote:npm:latest` script
 
-- Promotes an NPM package.
+- Promotes an NPM package to `latest`.
```

Comment:
Another random question but can anyone trigger a release ? Does it need special authentification ? I looked but am not able to find the code for it.

Reply:
> can anyone trigger a release ? Does it need special authentification ?

Anybody who is a collaborator to the repo is allowed to trigger the release workflow from [Actions > Create release](https://github.com/coveo/ui-kit/actions/workflows/release.yml).

Additionally, the workflow is run in the "Release" environment as you can see in `release.yml`, which requires approval immediately after triggering the workflow.
<img width="792" alt="image" src="https://github.com/coveo/ui-kit/assets/54454747/e7246657-03f0-4ccb-a0cf-060364c07e21">

Reply:
But what about the publish:npm:release in @atomic ? Would calling this start a release process for all of ui-kit ?

Reply:
I'm removing the publish:npm scripts currently. Prior to this PR, it would've only published the current version from all `package.json` files to NPM.